src/components/KonnectorUpdateInfo/styles.styl

```diff
@@ -1,4 +1,31 @@
+@require 'settings/breakpoints'
+
+.KonnectorUpdateInfo
+    padding-bottom 0
+
+    +small-screen()
+        padding-left 0.5rem
+        padding-right 0.5rem
+
+.KonnectorUpdateInfo__inner
+    padding 1.5rem
+    background-color var(--chablis)
+    border-radius 6px
+    text-align left
```

Comment:
Isn't it the default ?

Reply:
The parent element has `text-align: center` so we need to reset it.